jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -890,6 +894,11 @@ private void releaseChunk()
             _chunk = null;
         }
 
+        private long getRemainingContent()
```

Comment:
Does this need to be calculated in the loop?   Can't we just work out the length once when the content is reset/prepared ?

Reply:
The tests fail if this is only calculated once in reset/prepare.